templates/README.hbs

```diff
@@ -1,4 +1,4 @@
-<!-- To make changes, edit templates/README.hbs, not README.md! -->
+<!-- To make changes, edit templates/README.hbs, not this file. -->
```

Comment:
This file is `templates/README.hbs`, and it is what should be edited, so the "not this file" bit doesn't make sense.

How about: "<!-- To make changes, edit templates/README.hbs, not README.md or docs/modules_overview.md!. -->"